.eslintrc.js

```diff
@@ -1,25 +1,24 @@
 var double = "double";
 
 module.exports = {
-  root: true,
-  env: {
-    browser: true,
-    node: true
-  },
-  parserOptions: {
+	root: true,
+	env: {
+		browser: true,
+		node: true
+	},
+	parserOptions: {
 	parser: 'babel-eslint'
-	// "parser": "babel-eslint"
-  },
-  extends: [
+	},
+	extends: [
 	//'eslint-config-fluid',
-    '@nuxtjs',
-    'plugin:nuxt/recommended'
-  ],
-  // add your custom rules here
-  rules: {
-	  "indent": [2, "tab"],
-	  "no-tabs": 0,
-	  "quotes": [2, "double", { "avoidEscape": true }],
-	  "import/no-named-as-default": 0
-  }
+		'@nuxtjs',
+		'plugin:nuxt/recommended'
+	],
+	rules: {
+		"indent": [2, "tab"],
```

Comment:
I'm not sure if this is not working or I'm misunderstanding something. It seems that you are using a single tab and not 2 tabs for indentation. Does the 2 mean something else?

Reply:
I believe you are right it should be one.

This change has been added